controllers/TasksController.php

```diff
@@ -27,4 +35,69 @@ public function actionIndex()
             ]
         );
     }
+
+    /**
+     * Страница просмотра задания
+     *
+     * @return string - код страницы просмотра задания
+     */
+    /**
+     * Показывает страницы просмотра задачи
+     *
+     * @return string
+     */
+    public function actionView()
```

Comment:
http://localhost:8080/index.php?r=task/view&taskId=50

http://localhost:8080/task/view/50
```php
'task/view/<taskId:[\d]>' => 'task/view',
```

```suggestion
    public function actionView(int $taskId)
```